gnosis/eth/clients/etherscan_client.py

```diff
@@ -102,6 +103,7 @@ class EtherscanClient:
         EthereumNetwork.KROMA_SEPOLIA: "https://api-sepolia.kromascan.com",
         EthereumNetwork.BLAST_SEPOLIA_TESTNET: "https://api-sepolia.blastscan.io",
         EthereumNetwork.FRAXTAL_MAINNET: "https://api.fraxscan.com",
+        EthereumNetwork.BASE: "https://api.basescan.org/api",
```

Comment:
```suggestion
        EthereumNetwork.BASE: "https://api.basescan.org",
```
`api` will be added here [api](https://github.com/safe-global/safe-eth-py/pull/886/files#diff-fc2ac3c034d4ca719d90e843a0289d70d13706367e38642d27040e4d05634d22R191)